cob_hardware_config/raw3-3/config/phidgets.yaml

```diff
@@ -3,13 +3,13 @@ update_mode: polling                #switch between poll and event driven proces
 
 boards:
   ifk_general:
-    serial_num: 285651
+    serial_num: 272849
     ratiometric: true
     sensors:
-      bat1:
+      voltage:
```

Comment:
well...before #400 was merged, raw3-3 was the only robot affected by this parameter name change...

@ipa-bnm can you please clarify whether the PR (in cob_voltage_control? in cob_phidgets?) already has been merged?

I would kindly ask @destogl to send another PR for raw3-5 once @ipa-bnm confirmed that this name change has made it to ipa320/indigo_dev branch